EmoPy/tests/unittests/dataloader_test.py

```diff
@@ -1,8 +1,4 @@
-import sys
-
-import pytest
```

Comment:
We can probably also remove pytest from the requirements if we're happy with these changes, but i thought I'd leave it for now as some libraries seem to use it...  Better safe than sorry!